types/datasource.d.ts

```diff
@@ -185,4 +186,15 @@ export declare class DataSource extends EventEmitter {
     args?: any[] | object,
     options?: Options
   ): Promise<any>;
+
+/**
+ * Begin a new Transaction calling the underlying connector's begin method.
```

Comment:
`underlying connector's begin method` - that's not an accurate description, is it? We call `Transaction.begin` that calls connector's method `beginTransaction`.

```suggestion
 * Begin a new transaction.
 ```

Reply:
That is correct - good catch!